lib/src/facts/aix/operating_system_resolver.cc

```diff
@@ -0,0 +1,68 @@
+#include <internal/facts/aix/operating_system_resolver.hpp>
+#include <internal/util/regex.hpp>
+#include <facter/facts/os.hpp>
+#include <facter/execution/execution.hpp>
+#include <facter/util/file.hpp>
+
+#include <sstream>
+#include <algorithm>
+
+using namespace std;
+using namespace facter::util;
+namespace execution = facter::execution;
+
+string getattr(string object, string field)
+{
+    string result;
+
+    // To get architecture, can/should we use getconf KERNEL_BITMODE, getconf HARDWARE_BITMODE?
+    execution::each_line(
+        "/usr/sbin/lsattr", {"-El", object, "-a", field},
```

Comment:
It looks like it might not be possible, based on the `lsattr` man page. The `-E` option displays effective values for a specific 'Customized Device' object class (specified with `-l`). Since we're checking both `proc0` and `sys0`, I'm not sure if we can get both to return in one call.